packages/grid/_modules_/grid/components/panel/GridColumnsPanel.tsx

```diff
@@ -16,28 +16,27 @@ import { GridPanelWrapper } from './GridPanelWrapper';
 import { GRID_EXPERIMENTAL_ENABLED } from '../../constants';
 import { useGridRootProps } from '../../hooks/utils/useGridRootProps';
 
-const useStyles = makeStyles(
-  {
-    container: {
-      padding: '8px 0px 8px 8px',
-    },
-    column: {
-      display: 'flex',
-      justifyContent: 'space-between',
-      padding: '1px 8px 1px 7px',
-    },
-    switch: {
-      marginRight: 4,
-    },
-    dragIcon: {
-      justifyContent: 'flex-end',
-    },
-  },
-  { name: 'MuiDataGridColumnsPanel' }, // TODO rename to MuiGridColumnsPanel
-);
+const GridColumnsPanelRoot = styled('div', {
+  name: 'MuiGridColumnsPanel',
```

Comment:
`{ name: 'MuiDataGridColumnsPanel' }, // TODO rename to MuiGridColumnsPanel`
are dependencies renamed too?

Reply:
What do you mean by dependencies? This is to match our naming convetion.